### DISCUSSION TITLE
"Light diode"

### DISCUSSION THREAD

**Discussion by jumpjack** [ricktu288/ray-optics#131] [Ideas]
Is it possible to make a blocker block the light in one direction but not in the other? Although not possible in real life, it would be useful in simulations.

**Comment by ricktu288** [comment]
Now it is possible to use a single-sided Tools -> Other -> Custom Surface without any outgoing rays
[Example](https://phydemo.app/ray-optics/simulator/#XQAAAAK0AAAAAAAAAABCKcrGU8hqLFnpmU9EnTFrVldP-XiP9Y4WtYuOC-ZDaTnwCj1gZMfSiIkSNXCfWnlz3h4hqJHwS5Jc4GBi9hqfCEgspN96fkCUrorGqGkk6guN5Xe-NwNmU9wH66ntJYjDwOD87MwY-EoyDNojkx6ULPTT_TD6SHYxeorxP__xqaAA)
